### PR TITLE
Add a parameter to executive summary calculations

### PR DESCRIPTION
**Pull request by MonikaFu**
With the new SBTi calculation we need to pass to executive summary the directory when pacta inputs are because that is where the file with SBTi data merged with FactSet data will live.

This depends on a PR in executive summary: https://github.com/RMI-PACTA/pacta.executive.summary/pull/346. This PR needs to be merged first for this one to work. Also, the `pacta-data` folder needs to contain `fin_data_net_zero_targets.csv` and has to use the most recent version of the testing `user_data` (https://github.com/RMI-PACTA/user_results/pull/24/files/f75b9955f640a74124003e524a6a5789d9b616b8..355ef24f21422238c88be9ad6ff571ef2d5e56db).

### CODE REVIEW

**Comment by MonikaFu** [comment]
@cjyetman done. Sorry but this is still in a draft mode so I was not worrying about the description yet.

**Review by jdhoffa — APPROVED** [review]

**Comment by cjyetman** [comment]
is this still expected?
https://pactadatadev.blob.core.windows.net/ghactions-workflow-transition-monitor-results-reports/rmi_pacta_2023q4_pa2024ch-20240702T105329Z/EN/assetmanager/1/working_dir/50_Outputs/rmi_pacta_2023q4_pa2024ch/executive_summary/template.pdf
Screenshot 2024-07-02 at 14 06 30

**Comment by MonikaFu** [comment]
> is this still expected? https://pactadatadev.blob.core.windows.net/ghactions-workflow-transition-monitor-results-reports/rmi_pacta_2023q4_pa2024ch-20240702T105329Z/EN/assetmanager/1/working_dir/50_Outputs/rmi_pacta_2023q4_pa2024ch/executive_summary/template.pdf Screenshot 2024-07-02 at 14 06 30

yes, this is related to the WEO 2023 scenarios issue

File: .github/workflows/build-push-private.yml [hunk 1]
```diff
@@ -107,6 +108,11 @@ jobs:
           echo "peer-results=$PEER_RESULTS"
           echo "peer-results=$PEER_RESULTS" >> "$GITHUB_OUTPUT"
 
+          # includes handling for null/missing keys
+          NET_ZERO_TARGETS="$(jq -r '.net_zero_targets | select( . != null )' $config_file)"
```
Comment (cjyetman) [inline .github/workflows/build-push-private.yml:112]: does this modify the data in some way that one would have to manually do if they were trying to replicate this locally?

**Review by cjyetman — APPROVED** [review]

File: build/config/rmi_pacta_2023q4_pa2024ch.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
   "project_code": "PA2024CH",
   "templates_ref": "",
   "peer_results": "https://pactadatadev.blob.core.windows.net/testing-files/peer-results/PA2024CH_peer-results-test",
+  "net_zero_targets": "https://pactadatadev.blob.core.windows.net/project-files/pa2024ch/fin_data_net_zero_targets.csv",
```
[the remark this answers is not in the document]
Comment (AlexAxthelm) [reply build/config/rmi_pacta_2023q4_pa2024ch.json:7]: from the pactadatadev" storage account, "Containers". (in the left side menu), "project-files", and follow t he directory structure from there.

**Review by AlexAxthelm — APPROVED** [review]

File: build/config/rmi_pacta_2023q4_pa2024ch.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (cjyetman) [reply build/config/rmi_pacta_2023q4_pa2024ch.json:7]: I don't have Containers in the left side menu?
Screenshot 2024-07-02 at 14 24 44

**Pull request merged**